packages/expo-image/src/Image.tsx

```diff
@@ -27,6 +27,7 @@ export interface ImageProps extends AccessibilityProps {
   source?: ImageSourcePropType | null;
   style?: StyleProp<ImageStyle>;
   resizeMode?: ImageResizeMode;
+  fadeDuration?: number;
```

Comment:
As long as it's Android only, let's mark it 😉 
```suggestion
  /**
   * @Android only
   */
  fadeDuration?: number;
```